### ISSUE TITLE
Needs additional i18n translations

### ISSUE THREAD

**Issue by imavroukakis** [republique-et-canton-de-geneve/chvote-1-0#4]
In order to catch as wide an audience as possible, internationalisation is imperative.

**Comment by chvote-etat-de-geneve** [comment]
If you mean i18n support for the application, it is already implemented by using the resource bundle mecanism and JavaFx standard i18n. The default application resource bundle is located here: [offlineadmin-messages.properties](https://github.com/republique-et-canton-de-geneve/chvote-1-0/blob/master/admin-offline/src/main/resources/ch/ge/ve/offlineadmin/bundles/offlineadmin-messages.properties). Other versions implementing other languages than French can be added.

**Comment by imavroukakis** [comment]
I do. Unfortunately it's failed to pick up my English locale and reverted to French.

Edit: Oh i see what you mean, only French is available. Might be a good idea for me to try and translate it then

**Comment by chvote-etat-de-geneve** [comment]
Indeed... Since the application has only ever been used in Geneva so far, no other language than French has been needed yet.
Translations for other locales are welcome, but will need to be verified before being included.
